Pocket/pocket-ios
# Fetch all tags on initial sync

## Summary
Adds logic to `FetchList` operation that will download all the user's tags on the initial sync (since subsequent tags will be sync'd as they're added).

## References 
https://getpocket.atlassian.net/browse/IN-866

## Implementation Details
- Add `TagsQuery` to Apollo codegen
- Adds some logic to paginate through all the user's tags in `FetchList`
- Updates `FetchListTests` to properly stub out requests for tags in all methods
- Update UI tests to stub out the request for tags as well

## Test Steps

Scenario: Tag shows up in tag filter menu
1. Create an tag with no associated items on a separate device
2. Log into the app for the first time
3. Navigate to My List
4. Tap "Tagged" in filter carousel
5. The tag created in Step 1 should be present

Scenario: Tag appears in Add Tag screen
1. Create an tag with no associated items on a separate device
2. Log into the app for the first time
3. Navigate to My List (make sure you have at least one item)
4. Tap overflow menu for item, Tap "Add tags"
5. The tag created in Step 1 should be present

## PR Checklist:
